config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-main.yaml

```diff
@@ -53,6 +53,44 @@ periodics:
     testgrid-tab-name: capi-e2e-main
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "2"
+- name: periodic-cluster-api-e2e-main-mink8s
+  interval: 1h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: master
+    path_alias: sigs.k8s.io/cluster-api
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210808-1eaeec7-1.21
+      args:
+      - runner.sh
+      - "./scripts/ci-e2e.sh"
+      env:
+      - name: GINKGO_SKIP
+        value: "\\[Conformance\\] \\[K8s-Upgrade\\]"
+      # This value determines the minimum Kubernetes
+      # supported version for Cluster API management cluster.
+      #
+      # We should use the latest Kubernetes patch version with a published image, please
+      # refer to https://hub.docker.com/r/kindest/node/tags?page=1&ordering=last_updated
+      - name: KUBERNETES_VERSION_MANAGEMENT
+        value: "v1.19.11"
```

Comment:
Q: should this be `stable-1.19` (if the kind image is not published, we are building it locally)

Reply:
@fabriziopandini This would roughly double the test execution time imho. I guess that's not too critical in periodic tests.

Which variant do you prefer?

Reply:
Lets use the release label, otherwise we will have to maintain the version label in sync.

If the duration is long of we have a problem in resources, we can always move periodic tests from running every hours to run every 2h (or 4h), that IMO it is still reasonable for the project needs

Reply:
Sounds good to me, fixed